### ISSUE TITLE
About the pre-trained PointNet AE

### ISSUE THREAD

**Issue by Wuziyi616** [ajhamdi/AdvPC#1]
Hi! Thank you for your great work! I have a small question about the code. In the attack.py file [here](https://github.com/ajhamdi/AdvPC/blob/master/attack.py#L255-L262), you load a pre-trained autoencoder. According to the program output, the pre-trained weight should be at "latent_3d_points/data/single_class_ae/models.ckpt-500". However, I can't find it in this repo, because there is no "data/" folder in "latent_3d_points/" folder.

So, I wonder if I need to train the AE by myself? Or, can you kindly share the pre-trained weight with me? Because training 500 epochs on ShapeNet is really time-consuming I guess. And just to make sure, is this AE the one you use in your attack framework? That is to say, you will apply the adversarial loss on the reconstructed examples from this AE to perform your highly transferable attack?

**Comment by ajhamdi** [comment]
The weights of the trained AE used in the attacks are now uploaded [here](https://github.com/ajhamdi/AdvPC/tree/master/latent_3d_points/data/single_class_ae) . The weights of some new AE (used for evaluation ) is also now uploaded [here](https://github.com/ajhamdi/AdvPC/tree/master/latent_3d_points/data/new_ae)
